src/uct/api/uct.h

```diff
@@ -2501,7 +2502,10 @@ typedef enum {
     UCT_MEM_ALLOC_PARAM_FIELD_MDS            = UCS_BIT(3),
 
     /** Enables @ref uct_mem_alloc_params_t::name */
-    UCT_MEM_ALLOC_PARAM_FIELD_NAME           = UCS_BIT(4)
+    UCT_MEM_ALLOC_PARAM_FIELD_NAME           = UCS_BIT(4),
+
+    /** Enables @ref uct_mem_alloc_params_t::sys_device */
```

Comment:
it is called `sys_dev` in `uct_mem_alloc_params_t`. Pls align one of them with the other.
also `UCT_MEM_ALLOC_PARAM_FIELD_SYS_DEVICE`

Reply:
UCT_MEM_ALLOC_PARAM_FIELD_SYS_DEV